src/tests/test_process.py

```diff
@@ -94,8 +143,40 @@ def _test_threads(self):
         #   - InvBroadcaster
         #   - AddrBroadcaster
         #   - Downloader
-        self.assertEqual(
-            len(self.process.threads()), self._threads_count)
+        #   - Uploader
+
+        self.longMessage = True
+
+        try:
+            thread_names = \
+                subprocess.check_output(["ps", "-L", "-o", "comm=",
+                                         "--pid",
+                                         str(self.process.pid)]).split()
+        # pylint: disable=broad-except
+        except BaseException:
+            thread_names = []
+
+        extra_threads = []
+        missing_threads = []
+        for thread_name in thread_names:
+            if thread_name not in self._threads_names:
+                extra_threads.append(thread_name)
+        for thread_name in self._threads_names:
+            if thread_name not in thread_names:
+                missing_threads.append(thread_name)
+
+        msg = "Missing threads: {}, Extra threads: {}".format(
+            ",".join(missing_threads), ",".join(extra_threads))
+
+        self.assertGreaterEqual(
+            len(self.process.threads()),
```

Comment:
`AccessDenied` on osx: https://travis-ci.org/github/g1itch/PyBitmessage/jobs/738524694

Reply:
Test is now skipped on OSX.